xkit.js

```diff
@@ -1074,7 +1074,7 @@ function xkit_init_special() {
 	}
 
 	if (document.location.href.indexOf("/xkit_editor") !== -1) {
-		if (XKit.browser().chrome === true) {
+		if (XKit.browser().chrome || XKit.browser().firefox) {
```

Comment:
should test for the presence of the chrome/browser global instead of manually checking for the extension.